hadoop-hdds/docs/content/interface/CSI.zh.md

```diff
@@ -0,0 +1,78 @@
+---
+title: CSI 协议
+weight: 3
+summary: Ozone 支持 容器存储接口(CSI) 协议。你可以通过Ozone CSI 挂载 Ozone 桶的方式使用 Ozone。
+---
+
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+容器存储接口 `Container Storage Interface` (CSI) 使存储供应商（SP）能够一次性开发一个插件，并让它跨多个容器编排工作。
+
+获取更多CSI的信息，可以参考[SCI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+
+Ozone CSI是 CSI 的一种实现，它使得在容器里使用Ozone成为了可能。 
+
+## 入门
+
+首先，我们需要一个带有S3Gateway的Ozone集群，并且它的OM和S3Gateway的端口都可以对CSI pod可见，
+因为CSIServer 将会访问 OM 来创建或者删除篮子，并且，CSIServer将会通过goofys创建到s3g的挂载点的方式来发布桶。 
+
+如果你没有一个运行在kubernetes上的Ozone集群，你可以参考[Kubernetes]({{< ref "start/Kubernetes.zh.md" >}}) 来创建一个。
```

Comment:
```suggestion
如果你没有一个运行在kubernetes上的Ozone集群，你可以参考[Kubernetes](https://github.com/apache/hadoop-ozone/blob/master/hadoop-hdds/docs/content/start/Kubernetes.zh.md) 来创建一个。
```

Reply:
@cxorm I don't think we should hard-code URLs like that.  What's the problem with the Hugo `ref` shortcode?

Reply:
Thank you @adoroszlai for the reminder.

Yeah the URL here could be not hard-code by Hugo. (I just show it in other .md editor and not consider Hugo)

No problem here.

Reply:
@cxorm https://ci-hadoop.apache.org/view/Hadoop%20Ozone/job/ozone-doc-master/lastSuccessfulBuild/artifact/hadoop-hdds/docs/public/interface/csi.html

Reply:
Thank you @maobaolong for the reply.
No problem here, it's my carelessness.